cla_backend/apps/reports/tests/test_translate_savings.py

```diff
@@ -0,0 +1,41 @@
+from unittest import TestCase
```

Comment:
The Django convention appears to be `from django.test import TestCase`

Reply:
that seems to bring in a database. There seem to be many simple examples that use this style when not dependant on Django